esmvaltool/recipes/recipe_autoassess_landsurface_permafrost.yml

```diff
@@ -0,0 +1,62 @@
+# ESMValTool
+# recipe_autoassess_landsurface_permafrost.yml
+---
+documentation:
+  description: |
+    Recipe that runs the Autoassess Land-surface assessment area diagnostic.
+    Land surface area and ice masks are time independent.
+    
+  authors:
+    - predoi_valeriu
+
+  maintainer:
+    - predoi_valeriu
+
+datasets:
+  - {dataset: MPI-ESM-LR,  project: CMIP5, exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+  - {dataset: MPI-ESM-MR,  project: CMIP5, exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+  - {dataset: inmcm4,      project: CMIP5, exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+
+preprocessors:
+  pp_aa_area:
+    regrid:  # NOT USED
+      target_grid: 0.15x0.15
+      scheme: linear
+
+diagnostics:
+  aa_landsurf_permafrost:
+    description: Autoassess test diag for Land-Surface Permafrost.
+    variables:
+      tas:
+        mip: Amon
+      tsl:
+        mip: Lmon
+      sftlf:
+        mip: fx
+      mrsofc:
```

Comment:
Actually, this should be mrsos (also in Lmon). The code wants the actual soil moisture content (=mrsos) whereas mrsofc is the soil moisture capacity.

Reply:
the CMIP6 CMOR table entry for `mrsos` says:
```
        "mrsos": {
            "frequency": "mon",
            "modeling_realm": "land",
            "standard_name": "mass_content_of_water_in_soil_layer",
            "units": "kg m-2",
            "cell_methods": "area: mean where land time: mean",
            "cell_measures": "area: areacella",
            "long_name": "Moisture in Upper Portion of Soil Column",
            "comment": "The mass of water in all phases in the upper 10cm of the  soil layer.",
            "dimensions": "longitude latitude time sdepth1",
            "out_name": "mrsos",
            "type": "real",
            "positive": "",
            "valid_min": "",
            "valid_max": "",
            "ok_min_mean_abs": "",
            "ok_max_mean_abs": ""
        },
        
 ```
are we sure that's what we need? I am changing the recipe now but that means changing the dig too

Reply:
another massive hurdle is that HadGEM is completely missing `mrsos` data from *all* experiments on JASMIN

Reply:
Yes, definitely. The code comments are consistent with this, and the original stash code m01s08i223 maps to mrsos in our CMIP6 submission.

Reply:
What's the "dig"?

Reply:
what dig?

Reply:
You said
> that means changing the dig too

I didn't know what that meant.

Reply:
sorry - typo, should be diag -> diagnostic :laughing:

Reply:
Haha - I should have worked that out.